src/buttons/FAB.tsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import { StyleSheet, Animated, StyleProp, ViewStyle } from 'react-native';
+import Button, { ButtonProps } from './Button';
+import { IconProps } from '../icons/Icon';
+import { FullTheme, withTheme } from '../config';
+
+export type FABProps = ButtonProps & {
+  iconName?: string;
```

Comment:
I would say that you can just use the `icon` prop that `Button` has instead of these 3 props.